sdk/lib/drivers/sound/mmixer/controls.c

```diff
@@ -221,7 +221,7 @@ MMixerAddMixerControl(
                 for(Index = 0; Index < Steps; Index++)
                 {
                     VolumeData->Values[Index] = Value;
-                    Value += Range->SteppingDelta;
+                    Value += Range->SteppingDelta - 1;
```

Comment:
Is this the thing that makes the left and right volume controls behave independently?
What's strange is that the SteppingDelta were to be 1, then now Value isn't basically increased.

Reply:
>Is this the thing that makes the left and right volume controls behave independently?

Yes, that's it.
>What's strange is that the SteppingDelta were to be 1, then now Value isn't basically increased.

As I remember from my previous debugging, the value of `Range->SteppingDelta` isn't `1` at first step, so subtracting `1` doesn't make it zero, and I've already tested: the volume level still increases correctly after this change. You can see how much `Value` increases on each step in debug output previously posted in the one of my comments below.